docs/ci-cd/generate_metrics.sh

```
@@ -0,0 +1,141 @@
+    
+    echo "| $count | $short_sha | $status | $duration | $status_label | $commit_msg |  |"
+}
+
+build_metrics_report() {
+    log "Generation started"
+    
+    write_table_header
+    
+    log "Fetching all runs data"
+
+    local runs_json=$(fetch_all_runs | jq -s '.')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)